src/microbe_stage/editor/PatchMapDrawer.cs

```diff
@@ -168,6 +553,9 @@ private void UpdateNodeSelections()
         {
             node.Selected = node.Patch == selectedPatch;
             node.Marked = node.Patch == playerPatch;
+
+            if (selectedPatch != null)
+                node.SelectionAdjacent = selectedPatch.Adjacent.Contains(node.Patch!);
```

Comment:
Aren't nodes always associated with a patch, so why is the nullability on patch needed to be overridden here?